src/main/resources/org/greencodeinitiative/java/ecoCode_way_profile.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
this file must be named "creedengo_way_profile.json" and check impact inside code where this file is mentionned
and the "name" property must also be updated